notes/style-guide.md

```diff
@@ -402,7 +402,7 @@ word within a compound word is capitalized except for the first word.
 
 * Rational variables are named `p`, `q`, `r`, ... (default `p`)
 
-* All other variables tend to be named `x`, `y`, `z`.
+* All other variables, including `Bool`, should be named `x`, `y`, `z`.
```

Comment:
Interesting choice. Why make it here? (Or: why are the numeric types special? isn't `Bool` also special?)

Does it belong as part of this PR? (It's clearly had lots of knock-on consequences throughout the PR)

Reply:
The numeric types are special, because they have special letters associated with them. I did think that `Bool` should use `a`, `b`, `c` etc, but then I remembered that we can't use `variable`s then because `a` those letters are already reserved for `Level`s. So I thought best to document this explicitly in the style-guide.

As for whether it belongs in the PR, it was a consequence of trying to use `variable`s in `Decidable.Core`, and then running into the above issue. I'm happy to separate it out if you would prefer?

Reply:
I think it would be better to legislate on such style points as a separate activity, so yes if it's easy to revert those parts of the PR, then please do...

As for `a` ... for `Level`s, I think that can be too dogmatic, especially when `\ell` is a (very) palatable alternative. And some modules seem much too promiscuous in reserving such names, and then not really using them, when Agda's level polymorphism in fact seems to permit 'overloaded' use of such variables.

And yes, I'd prefer `b` for `Bool`eans, but I think the balance between rigid style rules, consistency, and cognitive efficiency/comfort is a tricky negotiation.  In among all the swirl of change in v2.0, I think this is worth putting out for separate discussion/design/implementation?

Reply:
Okay, I removed the changes.